db/migrate/20220201101037_remove_initially_indexed_from_vacancies.rb

```diff
@@ -0,0 +1,6 @@
+class RemoveInitiallyIndexedFromVacancies < ActiveRecord::Migration[6.1]
+  def change
+    remove_index :vacancies, :initially_indexed
```

Comment:
This happens automatically.

> Indexes and table constraints involving the column will be automatically dropped as well

https://www.postgresql.org/docs/12/sql-altertable.html

Reply:
Ah cool – I think it's still worth having in there though in order for the migration to be cleanly reversible (not really a concern in this case as I can't think of a reason we would want to)!

Reply:
Good call, though in that case you'd also need to provide the type, null, defaults etc so it can be recreated.  You can pass `index: true` along with that, but at that point it's a style distinction rather than anything actually important :)